newIDE/app/src/MainFrame/EditorContainers/HomePage/CommunitySection/UserAndGameLeaderboards.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import * as React from 'react';
+import { UserFeedbackLeaderboard } from '../../../../CommunityLeaderboards/UserFeedbackLeaderboard';
+import { GameFeedbackLeaderboard } from '../../../../CommunityLeaderboards/GameFeedbackLeaderboard';
+import { ResponsiveLineStackLayout } from '../../../../UI/Layout';
+import { useResponsiveWindowSize } from '../../../../UI/Responsive/ResponsiveWindowMeasurer';
+import Paper from '../../../../UI/Paper';
+import Text from '../../../../UI/Text';
+import { Line, Spacer } from '../../../../UI/Grid';
+import { CommunityLeaderboardsContext } from '../../../../CommunityLeaderboards/CommunityLeaderboardsContext';
+
+const styles = {
+  leaderboardContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+  },
+};
+
+export const UserAndGameLeaderboards = () => {
+  const {
+    fetchCommunityLeaderboards,
+    gameLeaderboards,
+    userLeaderboards,
+  } = React.useContext(CommunityLeaderboardsContext);
+
+  React.useEffect(
+    () => {
+      fetchCommunityLeaderboards();
+    },
+    [fetchCommunityLeaderboards]
+  );
```

Comment:
```suggestion
  React.useEffect(
    fetchCommunityLeaderboards,
    [fetchCommunityLeaderboards]
  );
```

Reply:
Won't work unfortunately as it will now return a promise